### ISSUE TITLE
hours-detail:  Let's change Labels. 

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#908]
@dannypernik  I suggest the following:

Title:  Name Lastname's hours pending approval 
Description:   Lets eliminate the by....  its a bit confusing. 
![image](https://user-images.githubusercontent.com/26234440/34371706-0f6e1068-ea95-11e7-9929-dd076980a490.png)

**Comment by dannypernik** [comment]
@MaxTattenbach "Firstname Lastname's hours pending approval" sounds like it's the volunteer's hours, not the admin's. I think this is getting a bit nitpicky. Please close if it's not an important issue for you.

**Comment by MaxTattenbach** [comment]
@dannypernik  per discussed. 

Let's eliminate the by....   as we are already filtering by admin. 

And let's change title to Hours Pending Approval for Admin name and lastname

**Comment by dannypernik** [comment]
@MaxTattenbach I made the change in hours-detail. Note I decided to say "(Admin) approved hours for (description)" instead of using grey "Approved by" after description. This keeps the same format as "(Volunteer) recorded hours for (description)" when showing admin's hours. Cleans up the code a bit and makes more sense IMO.

Also changed titles to say "pending approval" on both hours-detail and org-admin.
